src/server/gateways/postgres/plugin.ts

```diff
@@ -74,26 +76,27 @@ export default class PostgresGateway extends Gateway {
       return response;
     } catch (err) {
       await client.query(TRANSACTIONS.ROLLBACK);
+      this.logger.log("error", "Company lookup failed");
       throw err;
     } finally {
       client.release();
     }
   }
 
-  public async readByDate(company: string): Promise<any> {
+  public async readByDate(date: string): Promise<any> {
```

Comment:
**Comment**: This `any` type should be removed and replaced with its correct value.

Reply:
Changed to use proper types.